app/src/test/java/org/freedombox/freedombox/View/Adapter/GridAdapterTest.kt

```diff
@@ -0,0 +1,67 @@
+/** This file is part of FreedomBox.
+ *
+ * FreedomBox is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * FreedomBox is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with FreedomBox. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.freedombox.freedombox.View.Adapter
+
+import com.google.gson.JsonArray
+import com.google.gson.JsonParser
+import kotlinx.android.synthetic.main.card.view.*
+import org.freedombox.freedombox.BuildConfig
+import org.freedombox.freedombox.Utils.ImageRenderer
+import org.freedombox.freedombox.Views.Adapter.GridAdapter
+import org.junit.Assert.assertEquals
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.RuntimeEnvironment.application
+import org.robolectric.annotation.Config
+
+
+@RunWith(RobolectricTestRunner::class)
+@Config(constants = BuildConfig::class)
+class GridAdapterTest {
+    val applicationContext = application.applicationContext
+    private val gridAdapter = GridAdapter(applicationContext, imageRenderer = ImageRenderer(applicationContext))
+    private var jsonArray = JsonArray()
+    private val jsonObject = JsonParser().parse("{'label': {'en': 'conversations'}, 'description': " +
```

Comment:
This looks awkward. Isn't there a way of creating a JsonObject from a Java HashMap?